alf/examples/ac_atari.gin

```diff
@@ -1,43 +1,58 @@
 # You need to install following packages
-# pip3 install atari-py opencv-python 
+# pip3 install atari-py opencv-python
 
 # environment config
-import tf_agents.environments.suite_atari
-create_environment.env_name='PongNoFrameskip-v4'
-create_environment.env_load_fn=@suite_atari.load
-create_environment.num_parallel_environments=30
-
-# AtariPreprocessing is not compatible with some version of gym. Make sure to
-# use gym==0.10.11.
-suite_atari.load.gym_env_wrappers=(@AtariPreprocessing, @FrameStack4)
+import tf_agents.environments.suite_gym
+import alf.environments.wrappers
+# From OpenAI gym wiki:
+# "v0 vs v4: v0 has repeat_action_probability of 0.25
+#  (meaning 25% of the time the previous action will be used instead of the new action),
+#   while v4 has 0 (always follow your issued action)
+# Because we already implements frame_skip in AtariPreprocessing, we should always
+# use 'NoFrameSkip' Atari environments from OpenAI gym
+create_environment.env_name='BreakoutNoFrameskip-v4'
```

Comment:
Since the current ac_pong.png is not the result of any of the configs, we should remove it or redo figures.

Reply:
Perhaps we should still keep a direct correspondence between config and result figure, so there is no confusion how the result is obtained.

Reply:
removed ac_pong.png; changed ac_atari.gin to be a common header